tests/testthat/test-renderer2-fill.R

```diff
@@ -0,0 +1,57 @@
+acontext("fill_off")
+
+#
+# test geom with both fill and colour styles
+#
+viz.point <- list(pointone = ggplot() + geom_point(
+  data = mtcars,
+  size = 10,
+  aes(x=wt, y=mpg,
+      colour = disp),
+  clickSelects = "gear")+
+  ggtitle("default alpha_off(0.5) style"),
+
+pointtwo = ggplot() + geom_point(
+  data = mtcars,
+  fill_off="transparent",
+  size = 10,
+  aes(x=wt, y=mpg,
+      fill = disp),
+  clickSelects = "gear")+
+  ggtitle("colour corresponding to `disp` group, fill_off=\"transparent\" "),
+
+pointthree = ggplot() + geom_point(
```

Comment:
seems like this test only covers geom_point. Do you think it would be worth testing another one like geom_rect? What about the geoms that are special cased in the JS code? (hline/vline/segment/linerange)

Reply:
Absolutely, I will add those tests. Thanks for the suggestion!